homeassistant/components/vizio/strings.json

```diff
@@ -4,19 +4,35 @@
         "step": {
             "user": {
                 "title": "Setup Vizio SmartCast Device",
+                "description": "All fields are required except Access Token. If you choose not to provide an Access Token, and your Device Type is 'tv', you will go through a pairing process with your device so an Access Token can be retrieved.\n\nTo go through the pairing process, before clicking Submit, ensure your TV is powered on and connected to the network. You also need to be able to see the screen.",
                 "data": {
                     "name": "Name",
                     "host": "<Host/IP>:<Port>",
                     "device_class": "Device Type",
                     "access_token": "Access Token"
                 }
+            },
+            "pair_tv": {
+                "title": "Complete Pairing Process",
+                "description": "Your TV should be displaying a code. Enter that code into the form and then continue to the next step to complete the pairing.",
+                "data": {
+                    "pin": "PIN"
+                }
+            },
+            "pairing_complete": {
+                "title": "Pairing Complete",
+                "description": "Your Vizio SmartCast device is now connected to Home Assistant.\n\nYour Access Token is '**{access_token_value}**'."
```

Comment:
We should not have to show access token. A user just wanted to pair the device, it will never need the access token again ?

Reply:
See comment above. I can update it so it only shows when a user has done a config import without their access token

Reply:
I have updated the text. This only shows on configuration.yaml import now